docs/getting_started/building_binaries.md

```diff
@@ -100,6 +101,7 @@ All properties have to be provided separatelly preceeded by "-D" -> property "sk
     * **jgroups**		(parameter `-Dno-jgroups` to disable)
     * **infinispan**		(parameter `-Dno-infinispan` to disable)
     * **process**		(parameter `-Dno-process` to disable)
+    * **redis**		(parameter `-Dno-redis` to disable)
```

Comment:
In my IDE the indentation doesn't look correctly at all for the whole paragraph. It renders fine in https://github.com/jmarkos/radargun/blob/eee318cf85d85d64edcea6ae8bbc43431c38337a/docs/getting_started/building_binaries.md